infra/src/dsl/value-files-generators/local-setup.ts

```
@@ -62,29 +70,32 @@ export const getLocalrunValueFile = async (
-        `cd "${rootDir}"`,
-        `. ./.env.${serviceNXName}`, // `source` is bashism
-        `echo "Starting ${name} in $PWD"`,
-        `yarn nx serve ${serviceNXName}`,
-      ],
+
+    if (serviceNXName) {
+      dockerComposeServices[name] = {
+        env: {
+          ...Object.entries(service.env)
+            .filter(shouldIncludeEnv)
+            .reduce((acc, [k, v]) => ({ ...acc, [k]: v }), {}),
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)